graylog2-server/src/main/java/org/graylog2/rest/resources/search/SearchResource.java

```diff
@@ -109,8 +109,12 @@ protected void validateInterval(String interval) {
         try {
             return searches.fieldStats(field, query, filter, timeRange);
         } catch (Searches.FieldTypeException e) {
-            LOG.error("Stats query failed. Make sure that field [{}] is a numeric type.", field);
-            throw new BadRequestException();
+            try {
+                return searches.fieldStats(field, query, filter, timeRange, true, true);
+            } catch (Searches.FieldTypeException e1) {
+                LOG.error("Stats query failed. Make sure that field [{}] is a numeric type.", field);
+                throw new BadRequestException();
```

Comment:
Adding a short descriptive message and maybe even the root cause would make sense here. Otherwise the user won't know what went wrong without looking into the Graylog server logs.